Integrations/FortiSIEM/FortiSIEM.py

```diff
@@ -0,0 +1,690 @@
+    object_info['groupId'] = group_id
+    object_info['active'] = True
+    object_info['sysDefined'] = False
+    demisto.info('\n\n{}\n\n'.format(object_info))
+
+    response = session.post(url, data=json.dumps(object_info), verify=VERIFY_SSL, auth=AUTH)
+    response = response.json()
+
+    if response.get('code', 0) == -1:
+        return_error(response['msg'])
+
+    return response
+
+
+def remove_item_from_resource_list_command():
+    args = demisto.args()
+    resource_type = parse_resource_type(args['resource_type'])
+    deleted_ids = args.get('ids', '').split(',')
+
+    raw_response = remove_item_from_resource_list(resource_type, deleted_ids)
+
+    return_outputs(raw_response, {}, raw_response=raw_response)
+
+
+def remove_item_from_resource_list(resource_type, deleted_ids):
+    session = login()
+    url = '{}/phoenix/rest/h5/{}/del'.format(HOST, resource_type)
+
+    response = session.delete(url, params={'ids': json.dumps(deleted_ids)}, verify=VERIFY_SSL, auth=AUTH)
+
+    if response.text != '"OK"':
+        return_error(response.text)
+
+    return 'items with id {} were removed.'.format(deleted_ids)
+
+
+def get_resource_list_command():
+    args = demisto.args()
+    resource_type = parse_resource_type(args['resource_type'])
+    group_id = args['group_id']
+
+    raw_response = get_resource_list(resource_type, group_id)
+    headers = raw_response.get('headerData', {}).get('keys', [])
+    ec = []
+    for element in raw_response.get('lightValueObjects', []):
+        e = dict(zip(headers, element.get('data', [])))
+        e['id'] = element.get('objectId')
+        ec.append(e)
+    outputs = {'FortiSIEM.Resource(val.id && val.id == obj.id)': createContext(ec, removeNull=True)}
+
+    return_outputs(tableToMarkdown('Resource list:', ec, headerTransform=pascalToSpace, removeNull=True),
+                   outputs,
+                   raw_response)
+
+
+def get_resource_list(resource_type, group_id):
+    session = login()
+    url = '{}/phoenix/rest/h5/{}/list'.format(HOST, resource_type)
+
+    params = {
+        'groupId': group_id,
+        'start': 0,
+        'size': 50,
+    }
+
+    response = session.get(url, params=params, verify=VERIFY_SSL, auth=AUTH)
+    response = response.json()
+
+    if response.get('code', 0) == -1:
+        return_error(response['msg'])
+
+    return response
+
+
+def prepare_args(d):
+    d = dict((k.replace("-", "_"), v) for k, v in d.items())
+    return d
+
+
+def test():
+    try:
+        login()
+    except Exception as e:
+        if isinstance(e, requests.exceptions.SSLError):
+            demisto.results("Not verified certificate")
+        else:
+            demisto.results(str(e))
+    demisto.results('ok')
+
+
+def fetch_incidents():
+    in_xml = create_query_xml("all", interval='1')
+    query_id = GetEventQuery(in_xml)
+    res = GetIncidentsByOrg(query_id)
+    known_ids = demisto.getLastRun().get('ids', None)
```

Comment:
it's limited to 1000 (line 642)
didn't find a better solution to not fetch duplicates
(the query results returns all incidents recently updated instead of recently created)